File: src/main/java/MainExecute.java
import NeuralNetwork.NeuralNetwork;
import NeuralNetwork.TestDataUtil;

import java.util.Arrays;

public class MainExecute {

    public static void main(String [] args) throws Exception {
        Float[][] inputsFromFile = TestDataUtil.readInputsFromFile("src/main/resources/inputs.txt");
        Integer[] outputsFromFile = TestDataUtil.readOutputsFromFile("src/main/resources/outputs.txt");

        int neurons = 10;
        int iterations = 10000;

        NeuralNetwork neuralNetwork = new NeuralNetwork(inputsFromFile, outputsFromFile);
        neuralNetwork.addHiddenLayer(neurons);
        neuralNetwork.train(iterations);

        Arrays.stream(inputsFromFile).forEach(input ->
                System.out.println(String.format("Predicted result for [%s,%s]:%s ", input[0], input[1], neuralNetwork.predict(new Float[] {input[0], input[1]}))));
    }

}

File: src/main/kotlin/MainExecute.kt
import neuralNetwork.NeuralNetwork
import neuralNetwork.TestDataUtil

import java.util.Arrays

object MainExecute {

    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val inputsFromFile = TestDataUtil.readInputsFromFile("src/main/resources/inputs.txt")
        val outputsFromFile = TestDataUtil.readOutputsFromFile("src/main/resources/outputs.txt")

        val neurons = 5
        val iterations = 50000

        val neuralNetwork = NeuralNetwork(inputsFromFile, outputsFromFile)
        neuralNetwork.addHiddenLayer(neurons)
        neuralNetwork.train(iterations)

        inputsFromFile.forEach { input -> println("Predicted result for [${input[0]},${input[1]}]:${neuralNetwork.predict(arrayOf(input[0], input[1]))} ") }
    }

}

